### ISSUE TITLE
Replace array does not replace the array

### ISSUE THREAD

**Issue by gbleu** [thomaspoignant/scim-patch#36]
A new array is pushed instead:

```
members [ { value: 'test1' } ]
operation { op: 'replace', path: 'members', value: [ { value: 'test2' } ] }
members [ { value: 'test1' }, [ { value: 'test2' } ] ]
```

**Comment by thomaspoignant** [comment]
Hi @gbleu, thanks for your issue.
I am gonna check the problem and comeback to you.

**Comment by thomaspoignant** [comment]
@gbleu the fix is available in the version **`v0.2.6`** of the library.
